app/assets/javascripts/custom.js

```diff
@@ -0,0 +1,3 @@
+$(function(){
```

Comment:
I figured since we don't have much js, I would just keep it generic for now, but I'll rename it.